packages/app/obojobo-repository/shared/components/dashboard.jsx

```diff
@@ -199,8 +209,18 @@ const renderModuleManageCollectionsDialog = (props, extension) => (
 	/>
 )
 
-const renderCollectionBulkAddModulesDialog = props => (
+const renderCollectionBulkAddModulesDialog = (props, extension) => (
 	<CollectionBulkAddModulesDialog
+		title=""
+		collections={props.myCollections}
+		selectedModules={props.selectedModules}
+		bulkAddModulesToCollection={props.bulkAddModulesToCollection}
+		onClose={extension.onClose}
+	/>
+)
+
+const renderBulkSuccessDialog = props => (
+	<BulkSuccessDialog
```

Comment:
The `BulkSuccessDialog` component doesn't appear to be using `collections`, `selectedModules` or `bulkAddModulesToCollection`. If they're not necessary to that component, it's not necessary (and cleaner) to not provide them to that component as props. I'd remove them here.